scripts/perftune.py

```diff
@@ -549,13 +552,48 @@ def _get_irqs(self):
         pass
 
 #### Private methods ############################
+    @property
+    def __ec2_metadata_base_url(self):
+        return "http://169.254.169.254/latest/"
+
+    @property
+    def __metadata_token(self):
+        """
+        Refresh IMDSv2 session token if it necessary, and return current token
+        :return: current session token
+        """
+        token_ttl = 21600
+        update_token = False
```

Comment:
`update_token` doesn't seem to be used for anything but assignment.
Unless I missed the place where you read this value, you should be able to safely drop this variable.